src/QsCompiler/Core/SyntaxGenerator.fs

```diff
@@ -35,17 +35,15 @@ and private StripPositionInfoFromNamespace(parent: StripPositionInfo) =
     inherit NamespaceTransformation(parent)
     override this.OnLocation _ = Null
 
-and public StripPositionInfo private (_internal_) =
+and public StripPositionInfo() as this =
```

Comment:
With this change, `StripPositionInfo()` now becomes a function that returns an instance of the class type `StripPositionInfo` and the `do` bindings are used to initialize the members inherited from `SyntaxTreeTransformation`? Am I understanding this change correctly?

Reply:
The API of StripPositionInfo is the same before and after the change - it's a class with a public parameterless constructor. I removed the additional private constructor that took a parameter named `_internal_`, because this constructor actually isn't necessary (I'm not sure why it was there to begin with).